src/lib/util.ts

```diff
@@ -77,6 +77,16 @@ export async function closeConnection(uuid: string) {
 	}
 }
 
+export function transformDiscordLocale(discordLocale: string): string {
+	if (discordLocale === "uk") discordLocale = "ua"
+	const localeWithCountryCode = discordLocale.replace(/([a-z]+)(?:-([A-Z]+))?/, (_, a, b) => (b ? a + b.toLowerCase() : a)),
+		locale = discordLocale.replace(/([a-z]+)(?:-([A-Z]+))?/, (_, a) => a)
+
+	if (readdirSync("./strings").includes(localeWithCountryCode)) return localeWithCountryCode
+	else if (readdirSync("./strings").includes(locale)) return locale
```

Comment:
see this is where you can make the result of readdirSync a constant

Reply:
Oh yeah you're right